plugins/Rockets/jsonSerialization.h

```diff
@@ -528,6 +544,42 @@ inline std::string to_json(const brayns::Scene& scene)
     return staticjson::to_json_string(scene);
 }
 
+template <typename T>
+inline std::string modelBinaryParamsToJson(const T& params)
+{
+    const auto replaceFirstOccurrence = [](std::string s,
```

Comment:
that looks suspiciously similar to jsonPropertyMap.h:802 ?!

Reply:
Moved to Utils.h